code/game/gamemodes/game_mode.dm

```diff
@@ -279,11 +279,18 @@
 			intercepttext += i_text.build(A)
 		else
 			intercepttext += i_text.build(A, pick(modePlayer))
-
-	print_command_report(intercepttext,"Centcom Status Summary")
-	priority_announce("Summary downloaded and printed out at all communications consoles.", "Enemy communication intercept. Security Level Elevated.", 'sound/AI/intercept.ogg')
-	if(security_level < SEC_LEVEL_BLUE)
-		set_security_level(SEC_LEVEL_BLUE)
+	
+	var/list/sec = get_living_sec()
+	for (var/obj/machinery/vending/security/S in machines)

```

Comment:
Why are you looping through sec vendors? This will cause multiple reports to be sent if there's more than one sec vendor on the map.